spec/ADAS/ADAS.vspec

```diff
@@ -83,33 +83,18 @@
   type: sensor
   description: Indicates if obstacle sensor system incurred an error condition. True = Error. False = No Error.
 
-- ObstacleDetection.DistanceToObject:
+- ObstacleDetection.Input:
```

Comment:
I guess it's no surprise that I question this one, first of all.  In #81 we discussed if an extra node is required in the YAML tree to hold the instance declarations or if those could be put directly under DistanceToObject.  I preferred not creating the extra node.  

However, I also thought we were converging in #81 towards that **if** an extra node is required in the YAML tree to hold the instance declarations , but **if** it ithen we seemed to talk about naming the node itself "instance" instead of an extra word like "Input"  which is only created because some name is needed (but later on, the idea of using a wildcard character like _ instead seemed to be considered also by @UlfBj ).

Furthermore, you just suddenly use the word "extension" instead of "instance" in the actual instance itself?  So are you then saying that you prefer this word over instance and is the intention that it shall mean the same thing?  I feel like you could explain/discuss that first.   (Is it something you have discussed in another forum which I missed?)

Reply:
@gunnarx: At the beginning when we were talking about the structure and we had "position" as a name, when I think @UlfBj requested to change it to extension. From back then I used the code, which I've created to experiment. So it slipped through. If it's requested I can call it instance.

> However, I also thought we were converging in #81 towards that if an extra node is required in the YAML tree to hold the instance declarations then we seemed to talk about naming the node itself "A.B.C.instance" instead of an extra proper word like "Input"

I guess I got lost in the discussions. I thought the request was to have it on branches only. It's doable :)